### DISCUSSION TITLE
Many Ideas and Suggestions

### DISCUSSION THREAD

**Discussion by Ranmyaku2** [Keriew/augustus#586] [Ideas]
I had a lot of Ideas and suggestions that maybe could be added or changed in the game. I love what you guys have done so far and I'm sure I'll love what you continue to do for the game and community. Seriously thank you for you contribution to this project it really is appreciated.

I had so many ideas instead of just posting a ton of them i attached a text file pick and chose any you think will benefit or none if they don't sound good. 
[Caesar 3 Augustus.odt](https://github.com/Keriew/augustus/files/8236036/Caesar.3.Augustus.odt)

**Comment by dmyersturnbull** [comment]
I really like these three suggestions (I only skimmed and I'm not one of the devs):
- The new campaign being in Rome :)
- Simply making the **Senate required for forums**. The campaign made it feel like this was intended.
- A **Beauty rating** (or Tourism? Could be `min{desirability-score, monument-score}` with feedback on what's holding it back.
- Stable that sends horses to the Barracks (and perhaps massively buffing cavalry).
- **More farm types with different fertile land types** (integrated with terrain features like rivers and mountains)
- **More trees and animal species**, preferably ones that move away when you build.
- More resources
- **Different types of natives**

### About the natives
- I found the native tribes mechanic annoying, with no strategic depth. You just remake the roads and plop down a couple of buildings. It's also a little absurd: Native people are totally fine with you taking their land if you tell them you're doing it. Simply **removing the Mission Post** might be a good change: You have to work *around* their land but want to be nearby for trading.

**Comment by thomasherena** [comment]
Not a dev, but I liked most of these, though I'm not sure about separating the market into three or four different types. I think maybe having different _types_ of markets _in addition_ to the vanilla might be better?

**Reply by Ranmyaku2** [reply]
true it might work well that way too. Withe the separating though you could do more with changing what certain class/house levels need hence the separating but either would work well.

**Comment by Ouaz** [comment]
40 pages of suggestions, impressive. :D

What I can say is that some ideas you suggest are planned (some are in an advanced state, others not so much ^^): custom campaign for Augustus, improved trade map, new resources (stone, sands, and herbs), irrigation system,  cart depot, farming reworking, herding animals, useful Senate,  etc.

Some of your other suggestions are pretty interesting, and can be considered.

Thanks for this big and neat contribution!
